cg/apps/tb/api.py

```diff
@@ -43,19 +41,22 @@ class TrailblazerAPI:
         AnalysisStatus.QC,
     ]
 
-    def __init__(self, config: dict):
-        self.service_account = config["trailblazer"]["service_account"]
-        self.service_account_auth_file = config["trailblazer"]["service_account_auth_file"]
+    def __init__(self, config: dict, keycloak_client: KeycloakClient):
+        self.keycloak_client: KeycloakClient = keycloak_client
+        self.keycloak_backend_user = config["trailblazer"]["keycloak_backend_user"]
+        self.keycloak_backend_user_password = config["trailblazer"][
```

Comment:
We should create a confidential client with a client secret. This should allow us to circumvent using username+password
<img width="699" alt="Screenshot 2025-05-13 at 14 15 28" src="https://github.com/user-attachments/assets/005b4c76-d5ea-47f9-ae7d-e5ef37dfb55f" />
rd